chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Script processes input files. If file exists - output it, otherwise
+# create it with permission 700 with bash64 password.
+set -eou pipefail
+
+if [ $# -lt 2 ]; then
+  echo "The script requires at least 2 arguments - filenames"
+  exit 1
+fi
+
+# Iterate thru files.
+while (( $# )); do
+  file="$1"
+  # If exists - show content.
+  if [ -e "${file}" ]; then
+    cat "${file}"
```

Comment:
It would be cool and more user friendly if you added filename above its contents

Reply:
Added "echo" before.